### ISSUE TITLE
Accessibility: Fingerprints are hard to read

### ISSUE THREAD

**Issue by erebion** [dino/dino#1665]
Fingerprints in colour are difficult to read, which is an accessibility issue. There's always a section I cannot read due to lack of contrast, whether in light mode or in dark mode.

Fingerprints have to be readable, so my suggestion would be to remove colour.

Please also keep in mind that for me this is only difficult, but a large part of humanity does perceive colour less or not at all.

Also, instead of having to verify fingerprints manually for devices that are not close, it'd be useful if Dino showed a QR code for those as well. I don't want to always take out a PC and boot it, just to scan the code. I know I have verified it, so why not show it for scanning? (Especially without https://github.com/dino/dino/issues/1449)

**Comment by dca00** [comment]
Yeah, the color-coding of fingerprints is quite cruel.

**Comment by fiaxh** [comment]
Fingerprints are actually not colored anymore on master 🙂 (Changed as part of 5ee322cbd987d36497522b82b876659e1d19da43 and 21ae42762d8a57da5cb1ec40b46e7510fc3121ad)
